tests/patterns/kotlin/single_interpolated_ident.kt

```diff
@@ -0,0 +1,8 @@
+
+val x = "whatever"
+
+// ERROR: match
+val x = "$FOO"
```

Comment:
Same comment as the `string_metavar` case. Also, I assume `$FOO` here isn't valid Kotlin because there's no variable `FOO` in scope? Not that our tests all need to be compilable but that's another place where it would help me understand what was going on

Reply:
And you can just apply this comment to `not_string_metavar` as well

Reply:
`var` is a keyword in kotlin, so i think in this case i'm just going to use an assigned-to variable